src/router/routes.js

```diff
@@ -15,6 +15,7 @@ export const RequestNewPassword = () =>  import(/* webpackChunkName: "newPwd-chu
 export const ResetPassword = () =>  import(/* webpackChunkName: "resetPwd-chunk" */ "@/views/Users/ResetPassword");
 export const EditProfile = () =>  import(/* webpackChunkName: "editProfile-chunk" */ "@/views/Users/EditProfile");
 export const OauthLogin = () =>  import(/* webpackChunkName: "Oauth-chunk" */ "@/views/Users/Login/OauthLogin.vue");
+export const Organisation = () =>  import(/* webpackChunkName: "Oauth-chunk" */ "@/views/Organisations/Organisation");
```

Comment:
You need to be carefull with the webpackChunkName comment which control the chunk name.
```suggestion
export const Organisation = () =>  import(/* webpackChunkName: "Organisation-chunk" */ "@/views/Organisations/Organisation");
```